app/code/community/Hps/Transit/Model/Payment.php

```diff
@@ -319,12 +331,12 @@ protected function closeTransaction($payment, $amount, $response, $status = self
 
         if (property_exists($response, 'avsResultCode')) {
             $payment->setCcAvsStatus($response->avsResultCode);
-            $details['avs_response_code'] = $response->avsResultCode;
-            $details['avs_response_text'] = $response->avsResultText;
+            $details['avs_response_code'] = $response->avsResponseCode;
+            $details['avs_response_text'] = $response->avsResponseMessage;
         }
 
-        if (property_exists($response, 'cvvResultCode')) {
-            $details['cvv_response_code'] = $response->cvvResultCode;
+        if (property_exists($response, 'cavvResponseCode')) {
+            $details['cvv_response_code'] = $response->cavvResponseCode;
```

Comment:
`cvv` and `cavv` represent two separate ideas. We need `cvv` in this case.

Reply:
updated the response code and message